src/install/haproxy/linux/install-tarball.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
+headingText: Download using tarball
+---
+
+    <Callout variant="tip">
+      We recommend [using your distribution package manager to install the infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/install-infrastructure-linux-using-package-manager). If you choose to [manually install the agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux), you may also want to install the on-host integrations from the provided tarballs.
+    </Callout>
+
+    The manual install process is not automated. If you opt for manual install, you must place the different files in the correct folders, and ensure that the agent has all the permissions to execute the integrations.
+
+    To install an integration from a tarball:
+
+    1. Download the packaged integration file from the [tarball repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/amd64/).
+    2. Unpack the tarball file according to our [integration file structure and placement rules](/docs/integrations/integrations-sdk/getting-started/integration-file-structure-activation), so the agent is able to find the definitions, configurations, and executables of the integration.
+    3. Place the binary that contains the definition file inside `newrelic-integrations` or `custom-integrations` in the [agent directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#agent-directory).
+    4. Place the integration's config file under the [plugin directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#configure-plugin).
+    5. Follow the JMX tool instructions below.
+
+       ### Use of the New Relic JMX tool [#nrjmx]
```

Comment:
oof this looks a little weird in the preview... It might not be worth the effort, but maybe we should have a branching path that's 'tarball with JMX' and 'vanilla tarball'

Reply:
i'll let you make the call tbh. i think austin would prefer this get shipped and fast followed later, so depends on how much lift this would need

Reply:
oh... i understand i missed the jmx tool in step 5. kinda glossed over it. Maybe this should be it's own mdx doc then?  like a different step 5 only for tarball